inc/profile.class.php

```diff
@@ -574,7 +574,7 @@ function showForm($ID, $options=array()) {
 
       $onfocus = "";
       $new     = false;
-      $rowspan = 5;
+      $rowspan = 4;
```

Comment:
No it's nor related to planning to, it's only to fix display with forget rowspan